dali/operators/expressions/expression_impl_cpu.h

```diff
@@ -28,26 +27,41 @@
 
 namespace dali {
 
-// Assumes that it will get a workspace for given Backend,
-// and the backend will store the inputs and outputs at given Backend.
-template <ArithmeticOp op, typename Result,
-          typename Left, bool LeftIsTensor,
-          typename Right, bool RightIsTensor>
-class ExprImplBinCPU : public ExprImplBase, ExprImplParam<CPUBackend> {
+template <ArithmeticOp op, typename Result, typename Input0>
```

Comment:
Why Input**0**?

Reply:
I thought that there may appear an 3-input operator, so it will probably have numbered inputs. That's why, this is also numbered. I can remove if you want. Especially as I left the Left and Right for binary.

Reply:
If binary has named, not numbered, inputs - then please, remove the 0.

Reply:
Done